packages/camera/camera_avfoundation/ios/Classes/FLTCam.m

```diff
@@ -838,6 +852,80 @@ - (void)resumePreviewWithResult:(FLTThreadSafeFlutterResult *)result {
   [result sendSuccess];
 }
 
+- (void)setDescriptionWhileRecording:(NSString *)cameraName
```

Comment:
I attempted that but since theres different things you have to do between calls when switching the camera vs initializing it seemed cleaner to leave completely separate. Maybe theres a better way you can find that I am missing